src/mce/provider_mce.cpp

```diff
@@ -116,14 +129,21 @@ ScreenNs::ScreenNs()
     set_blanked_ = setter(prop);
 }
 
-void ScreenNs::set_blanked(bool v)
+void MceNs::reset_properties()
 {
-    set_blanked_(v ? "1" : "0");
+    setProperties(defaults_);
 }
 
-void MceNs::reset_properties()
+KeyboardNs::KeyboardNs()
+    : Namespace("maemo_InternalKeyboard")
```

Comment:
@thp correct. As also because hierarchical namespace support is not added to statefs yet, so maemo.* or maemo/* is converted to maemo_ inside.